### PR TITLE
Card openers

### PR DESCRIPTION
**Pull request by Excellify**
Implements card openers for emotes, usernames and badges.
Only for twitch versions of theses.

### CODE REVIEW

**Comment by Excellify** [comment]
Believe this is ready for merge. Cards for 3rd party badges and emotes can wait as they most likely have to open their own card and not the internal twitch one

**Comment by AnatoleAM** [comment]
sussy

File: src/composable/useCardOpeners.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		});
+	}
+
+	function badgeClick(e: MouseEvent, badge: Twitch.ChatBadge) {
+		if (!nameClick(e)) return;
+		tools.setViewerCardPage({
+			type: 2,
+			selectedBadgeID: badge.id,
+		});
+	}
```
Comment (AnatoleAM) [inline src/composable/useCardOpeners.ts:36]: Wouldn't calling any of those functions cause an exception throw if `tools` hasn't been populated?

Comment (AnatoleAM) [reply src/composable/useCardOpeners.ts:36]: Oh nvm they're no-ops. That's fine then

**Review by AnatoleAM — APPROVED** [review]

**Pull request merged**